Homework 2.ipynb

```diff
@@ -50,11 +50,39 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    tst_mean = test_sample.mean()[0]
</pre><p>No need to print this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/edoson/Python-ML---Lesson-2/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>